src/styles/BookList.css

```diff
@@ -25,12 +27,13 @@
 }
 
 .list-item {
```

Comment:
- [ ] Let's work on some details in the book details section. :+1: 
  - Kindly make sure the book category(i.e. Fiction) is using `Montserrat` font family. 🧐️
  - Every other text in this section is using the `RobotoSlab` font family.  ✍️ 
  - Please remove the filling on the remove button to match the design. 🤓️
  - Please make sure you are using the correct color for the text. :+1: 

| Your design                                                             | Zeplin design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/224475420-4007a41f-e387-437b-9793-4b20132d49c5.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/224475419-6ab11b48-74ec-4717-9212-cb189483501a.png" width="300">|